src/assets/Timecard.scss

```diff
@@ -3,3 +3,61 @@
     margin-top: 20px;
   }
 }
+
+@media only screen and (max-width: 640px) {
+  .mobile-link-group {
```

Comment:
This styling is used to correct the nav links and can actually be applied regardless of the screen dimensions. I'd suggest giving it more semantic name moving it outside of the media rule